index.html

```diff
@@ -276,6 +277,44 @@ <h3 class="connect-text">LET'S CONNECT</h3>
   </div>
 </section>
 
+<!-- Contact Section Page-->
+<section id="contact" class="contact-sectionbro">
+  <div class="contact-container">
+    <h2 class="contact-header">Contact Me</h2>
+    <p class="contact-paragraph">
+      If you have an application you are interested in developing, a feature
+      that you need built or a project that needs coding. I’d love to help
+      with it
+
+    </p>
+    <form action="" method="post" id="forms class="form-inputs">
+       <input class="text-input"
```

Comment:
- [ ] [REQUIRED] Good job so far! `Form` should use the `action` and method attribute to integrate with [formspree service](https://formspree.io/html), `action` value should be similar to `https://formspree.io/f/{form_id}` 🙏🏼

Reply:
Hi @mukhammadrizooff thank you  
great work , i made the changes you requested